### PR TITLE
OCPBUGS#14864: Add known issue for Selinux relabeling issue for 4.12

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.12 only

Issue: https://issues.redhat.com/browse/OCPBUGS-14864

Link to docs preview: https://61568--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @gcharot, @chao007, @sferich888, @xltian, @mwerner2113, @gnufied

### CODE REVIEW

**Review by jeana-redhat — COMMENTED** [review]
nit, otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by gcharot** [comment]
Please add @jsafrane as reviewer

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -3213,6 +3213,8 @@ There is currently no workaround to resolve this issue if encountered. To avoid
 
 * Currently, a certificate compliance issue, specifically outputted as `x509: certificate is not standards compliant`, exists when you run the installation program on macOS for the purposes of installing an {product-title} cluster on VMware vSphere. This issue relates to a known issue with the `golang` compiler in that the compiler does not recognize newly supported macOS certificate standards. No workaround exists for this issue. (link:https://issues.redhat.com/browse/OSDOCS-5694[*OSDOCS-5694*])
 
+* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```
[the remark this answers is not in the document]
Comment (gcharot) [reply release_notes/ocp-4-12-release-notes.adoc:3216]: Can you also link to https://access.redhat.com/solutions/6906261 (this is for ODF customers)
thanks!

**Comment by lpettyjo** [comment]
> Please add @jsafrane as reviewer

Done!

**Comment by gcharot** [comment]
lgtm

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (gcharot) [reply release_notes/ocp-4-12-release-notes.adoc:3216]: @lpettyjo in case you missed my comment

[the remark this answers is not in the document]
Comment (lpettyjo) [reply release_notes/ocp-4-12-release-notes.adoc:3216]: Done.

**Comment by xltian** [comment]
lgtm

**Comment by kalexand-rh** [comment]
LGTM

**Pull request merged**